test/unit/decorators/grpc.decorator.unit.ts

```diff
@@ -0,0 +1,56 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: loopback4-extension-starter
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+import {expect} from '@loopback/testlab';
+import {grpc} from '../../..';
+import {Application} from '@loopback/core';
+import {GrpcBindings} from '../../../src/keys';
+import {
+  GreeterInterface,
+  HelloRequest,
+  HelloReply,
+} from '../../protos/greeter.proto';
+import {Reflector} from '@loopback/context';
+
+describe('@rpc decorator', () => {
+  it('defines reflection metadata for rpc method', () => {
+    class Greeter implements GreeterInterface {
+      @grpc()
+      SayHello(request: HelloRequest): HelloReply {
```

Comment:
Nice! This is what I was looking for in my previous comment. Looks like the implementation is good but the README needs an update?

Reply:
yes the readme is the one outdated

Reply:
Done I updated the docs and addressed the rest of the comments, I'm landing this PR now :)